index.html

```diff
@@ -25,7 +25,17 @@ <h1 id="logo">Aziz Mejri</h1>
           </li>
         </ul>
       </nav>
-    </header>
+      <div id="toggle-menu" class="hidden">
+        <div class="close-wrapper clearfix"><img src="./media/svg/close.svg" alt="Close"  onclick="toggleMenu()"/></div>
+        <nav class="toggle-nav">
+            <ul class="nav-list">
+                <li class="nav-item"><a href="#" onclick="mobileMenuOption(event,'Portoflio')">Portoflio</a></li>
+                <li class="nav-item"><a href="#" onclick="mobileMenuOption(event,'About')">About</a></li>
+                <li class="nav-item"><a href="#" onclick="mobileMenuOption(event,'Contact')">Contact</a></li>
+            </ul>
+        </nav>
```

Comment:
- [ ] Please know that clicking on the items of the mobile menu raises some JavaScript errors:
![image](https://user-images.githubusercontent.com/20567503/175571458-9815fe97-41f8-45e0-a08a-c1bb1807ebf4.png)
After some debugging I noticed that the ids you're giving to `mobileMenuOption` are incorrect 🙃🙃, for example, the first line should be like this: `<a href="#" onclick="mobileMenuOption(event,'projects')">Portoflio</a>`  
Kindly change the 3 of them to the correct ids so the menu works correctly 😁😁.